rust/xaynet-server/src/state_machine/phases/unmask.rs

```diff
@@ -74,7 +73,7 @@ impl Phase for PhaseState<Unmask> {
                 .s3
                 .upload_global_model(&key, &global_model)
                 .await
-                .map_err(|err| PhaseStateError::SaveGlobalModel(err))?;
+                .map_err(PhaseStateError::SaveGlobalModel)?;
```

Comment:
ok wow also didn't know that this is possible 😅

Reply:
that was one of the suggestions by clippy